tests/test_verifylib.py

```diff
@@ -969,10 +979,14 @@ def setUpClass(cls):
         # dump expired layout
         layout = copy.deepcopy(layout_template)
         layout.signed.expires = (
-            datetime.today() + relativedelta(months=-1)
+            datetime.datetime.now() + relativedelta( months=-4)
```

Comment:
This change is out of scope for this PR.

Reply:
And we probably don't need to change this for the the < to <= change made in this PR.